### DISCUSSION TITLE
What happend with the grey scroll bar?

### DISCUSSION THREAD

**Discussion by StormYudi** [wailsapp/wails#2678] [Q&A]
Hi community, 
I developed a wails app recently, I sent this app to windows users, and they can see a grey scroll bar on the right of the window, but in mac, everything is fine, how can I debug this? idk what is that? 
I open dev tool, it shows me that the scroll bar is not belongs to the webview.

**Windows:**
<img width="1223" alt="image" src="https://github.com/wailsapp/wails/assets/19419186/dd1168f4-ccb0-4c4d-a4ab-b8c497452f50">

**Mac:**
<img width="1054" alt="image" src="https://github.com/wailsapp/wails/assets/19419186/ac4a9136-60d7-4e87-8031-8a8f5eaeffbb">

**Comment by leaanthony** [comment]
Created an issue for this 👍 
https://github.com/wailsapp/wails/issues/2680
